set-based-validation/README.md

```diff
@@ -0,0 +1,27 @@
+# set-based-validation-axon
+A question that pops up a lot is How can I check if an account with a particular email address already exists? 
+This command side validation could be done with a small look-up table which can be queried during validation.
+
+You can use several ways to validate the set and in this repository you can find a few of them
```

Comment:
```suggestion
# Set Based Consistency Validation
A type of question that occurs a lot is "How can I check if an account with a particular email address already exists?". 
In essence this signals the requirement to be able to validate a set, which is a more intricate process when it comes to applications which follow the CQRS paradigm. Without going in full detail here why this is a problem, a straightforward solution is to enhance the command model by adding a small look-up table which can be queried during such a validation step.

There are several ways to invoke this set validation when using Axon, several of which are shared in this repository.
```

Reply:
Once the blog is online, I think we should reference it in the `README.md` too. Not something to do right now of course, just sharing it as a reminder.